src/Thread/Index.js

```diff
@@ -108,20 +94,35 @@ class Index extends Component {
     this.hideDeletePostDialog()
   }
 
-  toggleLocked() {
+  toggleLocked = () => {
     const { submit } = this.props
     const { key: threadKey, value: thread = {} } = this.props.thread || {}
     submit(toggleThreadLocked({ threadKey, thread }))
     this.hideLockDialog()
   }
 
-  toggleUpvote(postKey) {
+  toggleUpvote = (postKey) => {
     const { submit } = this.props
     const { value: posts = [] } = this.props.threadPosts || {}
     const user = this.props.authenticatedUser
-    const post = find(posts, (threadPost) => {
+    const postIndex = posts.findIndex( (threadPost) => {
```

Comment:
Should still use lodash's `find` here for non-ES6 compatible browsers (IE 11 basically).